fru-area.H

```diff
@@ -26,7 +26,7 @@ class ipmi_fru
         std::string iv_name;
 
         // Length of a specific fru area.
-        size_t  iv_len;
+        uint32_t  iv_len;
```

Comment:
I don't think you need to change these: size_t is not 8 bits; it's definitely big enough.